src/meteofrance/model/forecast.py

```diff
@@ -38,6 +38,31 @@ def probability_forecast(self) -> list:
         """Return the wheather event forecast."""
         return self.raw_data["probability_forecast"]
 
+    @property
+    def today_forecast(self) -> dict:
+        """Return the forecast for today."""
+        return self.daily_forecast[0]
+
+    @property
+    def nearest_forecast(self) -> dict:
+        """Return the nearest hourly forecast."""
+        now_timestamp = datetime.utcnow().timestamp()
+        sorted_forecast = sorted(
+            self.forecast, key=lambda x: abs(x["dt"] - now_timestamp)
+        )
+        return sorted_forecast[0]
+
+    @property
+    def current_forecast(self) -> dict:
+        """Return the forecast of the current hour."""
+        current_hour_timestamp = (
+            datetime.utcnow().replace(minute=0, second=0, microsecond=0).timestamp()
+        )
+        return next(
+            (x for x in self.forecast if x["dt"] == current_hour_timestamp),
+            None,
+        )
```

Comment:
simpler than next, you have:
`forecast_by_datetime ={item["dt"]:item for item in self.forecast}`

Then `return forecast_by_datetime.get(current_hour_timestamp)`

Reply:
OK taken into account to have a better code coverage.